app/main/views.py

```diff
@@ -87,25 +85,26 @@ def update_service(service_id):
     db.session.add(service)
     db.session.commit()
 
-    return jsonify(services=jsonify_service(service)), 200
+    return jsonify(services=jsonify_service(service)), http_status
```

Comment:
I don't have a strong opinion on this either way - when we were going to be generating a new ID it made sense to me to return the created artefact, but I'm not sure now if clients will do anything with the response?  If not, then it seems just returning a status code might be all that's needed.